voice/voice-ai/x/Actions/ActionsView.swift

```diff
@@ -197,21 +199,32 @@ struct ActionsView: View {
     func handleOtherActions(actionType: ActionType) {
         switch actionType {
         case .reset:
-            SpeechRecognition.shared.reset()
+            DeepgramASR.shared.reset()
```

Comment:
reset calls start, so it doesn't need to call setup again (which duplicates some steps). There is some issue with the current `reset` since it tries to disconnect the webscoket and reconnecting it at the same time (sending two messages concurrently), so it is not working as intended (sometimes need to `reset` twice to get ASR resuming again). This can be fixed in `reset` function by calling `start` after disconnect is confirmed, and let the speech synthesiser say greeting again after it is done